third_party/systemlibs/mkl_dnn.BUILD

```diff
@@ -0,0 +1,65 @@
+licenses(["notice"])  # BSD/MIT-like license
+
+filegroup(
+    name = "LICENSE",
+    visibility = ["//visibility:public"],
+)
+
+
+HEADERS = [
+  "dnnl_config.h",
+  "dnnl_debug.h",
+  "dnnl.h",
+  "dnnl.hpp",
+  "dnnl_types.h",
+  "dnnl_version.h",
+  "mkldnn_config.h",
+  "mkldnn_debug.h",
+  "mkldnn_dnnl_mangling.h",
+  "mkldnn.h",
+  "mkldnn.hpp",
+  "mkldnn_types.h",
+  "mkldnn_version.h",
+]
+
+genrule(
+    name = "link_headers",
+    outs = HEADERS,
+    cmd = """
+      for i in $(OUTS); do
+        i=$${i##*/}
+        ln -sf $(INCLUDEDIR)/mkl-dnn/$$i $(@D)/$$i
+      done
+    """,
+)
+
+cc_library(
+    name = "mkl_headers",
+    hdrs = HEADERS,
+    includes = ["."],
```

Comment:
when you remove the genrule above, you should remove hdrs= and includes=. bazel automatically allows all system include headers